oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/LucenePropertyIndex.java

```diff
@@ -232,6 +237,15 @@ public LucenePropertyIndex(IndexTracker tracker, ScorerProviderFactory factory,
         this.tracker = tracker;
         this.scorerProviderFactory = factory;
         this.augmentorFactory = augmentorFactory;
+        logConfigsOnce();
+    }
+
+    private void logConfigsOnce() {
```

Comment:
Why not just use a static block "static {" ? It is simpler, and does the same AFAIK.

Reply:
CACHE_FACET_RESULTS  is not static now, so Can't be use in static context in static block and making it static leads to single instantiation of variable and hence we won't be able to test both cached and uncached facets,